src/scripts/domain/moveCalculator.ts

```
@@ -126,4 +126,199 @@ export class MoveCalculator {
+
+      this.addPawnAttackMovesRec(board, moves, piece, path, toRemove, pos);
+
+      path.pop();
+      toRemove.pop();
+
+      pos.sub(direction).sub(direction);
+    }
+  }
+
+  private static addKingAttackMoves(
+    board: BoardInfo,
```

ESLint

[error] 'board' is defined but never used.

(@typescript-eslint/no-unused-vars)